sdk/identity/Azure.Identity/README.md

```diff
@@ -214,4 +223,5 @@ This project has adopted the [Microsoft Open Source Code of Conduct][code_of_con
 [eventhubs_client_library]: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/eventhub/Azure.Messaging.EventHubs
 [azure_core_library]: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/core/Azure.Core
 [identity_api_docs]: https://azure.github.io/azure-sdk-for-net/identity.html
+[identity_samples]: https://docs.microsoft.com/en-us/azure/active-directory/develop/sample-v2-code
```

Comment:
We will be adding samples for using the Azure.Identity library soon, but for now I don't want to point people directly to msal samples. I would rather omit the link until we have samples for the Azure.Identity library. So could you delete this link and the reference to it above. Or we could add Comming Soon to the ref above rather than a link.

Reply:
Okay, for the moment I have removed the link.